### DISCUSSION TITLE
AudioGridder issue

### DISCUSSION THREAD

**Discussion by doctormidi** [apohl79/audiogridder#513] [Q&A]
Okk..I use studio one and Logic Pro x.
I try all the format on a studio One.
VST3 : I try Kontakt it open on the server windows System and on the plugin on the Mac,but no audio Chanel is opened as usual when we open any vst instruments ,so no audio signal and the plugin don’t receive midi info.and no multi out option.
AU : also I try kontakt it open an audio Chanel but also no midi info so no audio signal and no multi out option.
VST : also the same but it open an audio Chanel and there is multi out option,but no midi signal and so no also an audio signal.
Thnx for your help..

**Comment by CaBuffalo** [comment]
Please try the new release Beta 2,

On Wed, Apr 21, 2021 at 4:30 PM doctormidi ***@***.***> wrote:

> Okk..I use studio one and Logic Pro x.
> I try all the format on a studio One.
> VST3 : I try Kontakt it open on the server windows System and on the
> plugin on the Mac,but no audio Chanel is opened as usual when we open any
> vst instruments ,so no audio signal and the plugin don’t receive midi
> info.and no multi out option.
> AU : also I try kontakt it open an audio Chanel but also no midi info so
> no audio signal and no multi out option.
> VST : also the same but it open an audio Chanel and there is multi out
> option,but no midi signal and so no also an audio signal.
> Thnx for your help..
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/513>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BPKUAGZ22VTW7AFUDTJ5NYNANCNFSM43LKCVCQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by melph50** [reply]
Hi - Same here on Studio One 5.  Drag the plugin to create an Instrument track works ok. Comminication between the client and server fine ie> I can select and instrument located on the server machine. However, there is no audio.

This is because in Studio one 5, when the plugin is inserted as an instruement, it should automaticly create a mixer fader. In Beta 2 version, this is not happpening. In beta 1, this worked ok.

**Reply by melph50** [reply]
Something broke in beta 2 with Studio One

**Reply by apohl79** [reply]
would you be able to do the same thing in b1 and b2 and for both send me the server and plugin logs? sounds like it has something to do with channel layouts, but there is no obvious change in the ag plugin that the daw should see. so the files will hopefully explain this. thx!

**Reply by melph50** [reply]
Will do - ok to post logs here?

**Reply by melph50** [reply]
OK, so Im new to this - where are the logs located?

**Reply by apohl79** [reply]
yes you can post them here. https://audiogridder.com/bug-report-diagnostics-locations/

**Reply by melph50** [reply]
OK - first two are for Beta 2 
[AudioGridderPlugin_2021-04-22_13-14-44.log](https://github.com/apohl79/audiogridder/files/6358141/AudioGridderPlugin_2021-04-22_13-14-44.log)
[AudioGridderServer_2021-04-22_13-10-27.log](https://github.com/apohl79/audiogridder/files/6358143/AudioGridderServer_2021-04-22_13-10-27.log)

**Reply by melph50** [reply]
Had to uninstall Beta 2 and reinstall Beta 1 - here are the remaining logs (beta 1)
[AudioGridderPlugin_2021-04-22_13-27-55.log](https://github.com/apohl79/audiogridder/files/6358201/AudioGridderPlugin_2021-04-22_13-27-55.log)
[AudioGridderServer_2021-04-22_13-27-20.log](https://github.com/apohl79/audiogridder/files/6358202/AudioGridderServer_2021-04-22_13-27-20.log)

**Reply by melph50** [reply]
Having checked Beta 1 again, yes when you make a new instrument channel with the plugin, it creates a fader in studio one mixer

**Reply by doctormidi** [reply]
Hello ..
1-AU Format Works On Logic Pro X But Without Multi Out Option,So same as Version 1.1.1
2-All Format On Studio One 4.6 and 5.2 dont works but now in 1.2 beta 2 there is a midi signal.
    Details :
1-VST3 don't open an audio Chanel,no multi out option, but there is a midi signal.
2-AU Format open an audio chanel,but there is no Midi signal, no audio signal and no multi out option.
3-VST Format don't open an audio Chanel,there is a midi signal, no audio signal, there is a multi out option but we can't select or add any Chanel cause the first audio Chanel don't appear.

thnxxxx for all your help.
